offchain/test-fixtures/src/docker_cli.rs

```diff
@@ -12,17 +12,23 @@
 
 use std::process::Command;
 
-pub fn build(dockerfile: &str, tag: &str, build_args: &[(&str, &str)]) {
+pub fn build(
+    dockerfile: &str,
+    tag: &str,
+    build_args: Option<&[(&str, &str)]>,
```

Comment:
I didn't get this modification. You could pass an empty array if you don't need extra arguments.

Reply:
What is the idiomatic way of dealing with optional parameters? I needed an optional `context` parameter, so I've made `build_args` an `Option<>` too to keep the function signature uniform (it is optional after all). It also makes the intent more explicit IMHO.

Reply:
So we don't wrap collection parameters with `Option<>`. Duly noted 👌🏽